2020-04-08 19:30:38,619 - ERROR -my_log.py -API_AUTO -30-日志信息：1222 
NoneType: None
2020-04-08 19:30:38,620 - ERROR -my_log.py -API_AUTO -30-日志信息：122223332 
NoneType: None
